cpp/data/system_updates/12.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -o errexit
+
+
+readonly HOSTNAME=$(hostname)
+if [[ $HOSTNAME == "ub16.uni-tuebingen.de" || $HOSTNAME == "ptah.ub.uni-tuebingen.de" ]]; then
+    if [ -e /var/spool/cron/root ]; then # CentOS
+        root_crontab=/var/spool/cron/root
+    else # Ubuntu
+        root_crontab=/var/spool/cron/crontabs/root
```

Comment:
This is correct for the servers, but e.g. on my local development machine this file will not exist because my root does not have a crontab. So i guess this would lead to an error when sed is called. 

I think it would be better to determine the OS depending on whether /etc/lsb-release (Ubuntu) or /etc/redhat-release (CentOS) exists. Then the path could be set according to the OS. If the crontab file does not exist yet, the replacing should be skipped.

Reply:
I think the simplest solution is to check the two locations and to not update if neither exists which is what I have done now.